src/plugins/data_views/public/data_views/data_views_api_client.ts

```diff
@@ -81,9 +84,7 @@ export class DataViewsApiClient implements IDataViewsApiClient {
    * Does a user created data view exist?
    */
   async hasUserDataView(): Promise<boolean> {
-    const response = await this._request<{ result: boolean }>(
-      this._getUrl(['has_user_index_pattern'])
-    );
+    const response = await this._request<{ result: boolean }>(HAS_DATA_VIEWS_PATH);
```

Comment:
This was previously hitting an undocumented public api instead of the internal api with the same functionality